ecmascript/transforms/react/src/refresh/mod.rs

```diff
@@ -790,15 +794,15 @@ impl<C: Comments> Fold for Refresh<C> {
             return module_items;
         }
 
-        module_items.visit_with(&Invalid { span: DUMMY_SP } as _, self);
+        self.visit_module_items(&module_items, &Invalid { span: DUMMY_SP } as _);
```

Comment:
Why did you change this? Seems identical to me.

Reply:
To save one import. I believe these two are identical.

Reply:
I see. Thank you!